src/app/CommandHandlerImpl.cpp

```diff
@@ -404,7 +404,8 @@ Status CommandHandlerImpl::ProcessCommandDataIB(CommandDataIB::Parser & aCommand
         Access::SubjectDescriptor subjectDescriptor = GetSubjectDescriptor();
         Access::RequestPath requestPath{ .cluster = concretePath.mClusterId, .endpoint = concretePath.mEndpointId };
```

Comment:
Don't we need to pass in the entityId here and in other places?  Looks like a few places like EventManagement were fixed to do that, but others were not.

It's unfortunate that we are using these structs with no way to force consumers to initialize the data, so the compiler is not catching this problem for us.  :(

Reply:
Maybe we should:

1. static_assert that `kInvalidCommandId`, `kInvalidAttributeId`, and `kInvalidEventId` are all equal.
2. Initialize `entityId` to that value they're all equal to (e.g. to `kInvalidAttributeId`).
3. Make sure that Check() fails out immediately if the request path has the invalid value.

That way at least at runtime we will hopefully check places that forget to set the entity id.

Reply:
I believe all Check calls have populated requestType and entityId.  The more clear requestType should make it clear what entityId represents now.  Let me know if you still feel that more is needed.

Reply:
This is resolved now.